rmgpy/rmg/main.py

```diff
@@ -801,9 +802,9 @@ def execute(self, **kwargs):
                                 logging.warn('Reaction thresholds/flags for Reaction System {0} was not updated due to resurrection'.format(index+1))
```

Comment:
Should this line also be indented? I think the arguments and closing parenthesis in the function calls above should also be indented.

Reply:
done